types/nightwatch/index.d.ts

```diff
@@ -1,11 +1,19 @@
-// Type definitions for nightwatch 1.3
+// Type definitions for nightwatch 2.0
 // Project: http://nightwatchjs.org
 // Definitions by: Rahul Kavalapara <https://github.com/rkavalap>
 //                 Connor Schlesiger <https://github.com/schlesiger>
 //                 Clayton Astrom <https://github.com/ClaytonAstrom>
 //                 Lukas Beranek <https://github.com/lloiser>
+//                 Vaibhav Singh <https://github.com/vaibhavsingh97>
```

Comment:
I need consent from the other users.

Reply:
I consent

Reply:
The new names are part of the @Nightwatchjs team at @BrowserStack. I will get them to 👍. I consent